moped-editor/src/views/projects/projectView/ProjectComponents/utils/componentList.js

```diff
@@ -20,3 +20,20 @@ export const useComponentListItemText = (component) =>
     listItemText.secondary = component.location_description;
     return listItemText;
   }, [component]);
+
+/**
+ * Identify if a component is map by checking all potential feature layers
+ * for features.
+ * @param {object} component - the moped_component object
+ * @returns {object} true if any layer has features else false
+ */
+export const useIsComponentMapped = (component) =>
+  useMemo(
+    () =>
+      component.feature_drawn_points?.length > 0 ||
+      component.feature_drawn_lines?.length > 0 ||
+      component.feature_intersections?.length > 0 ||
+      component.feature_signals?.length > 0 ||
+      component.feature_street_segments?.length > 0,
+    [component]
+  );
```

Comment:
this feels like a clear over-use of useMemo: we don't need a stable reference to this value (it's Boolean) and the computation is not expensive. i'm going to remove the memo.